crates/libp2p-networking/src/network/behaviours/dht/mod.rs

```diff
@@ -190,7 +190,7 @@ impl<K: SignatureKey + 'static> DHTBehaviour<K> {
         if let Some(entry) = kad.store_mut().get(&key.clone().into()) {
             // The key already exists in the cache
             if chan.send(entry.value.clone()).is_err() {
-                error!("Get DHT: channel closed before get record request result could be sent");
```

Comment:
This one seems like a real error.  If this chan is closed I think the network node is shutdown

Reply:
I believe we see this on `Canceled` requests when we do the outbound query deduplication but I'll make sure before merging